.github/workflows/release-python.yml

```diff
@@ -336,25 +330,30 @@ jobs:
         uses: PyO3/maturin-action@v1
         with:
           command: build
-          target: ${{ steps.target.outputs.target }}
+          target: ${{ matrix.target }}
           args: >
             --profile dist-release
             --manifest-path py-polars/runtime/${{ matrix.package }}/Cargo.toml
             --out dist
-          manylinux: ${{ matrix.architecture == 'aarch64' && '2_24' || 'auto' }}
+          manylinux: ${{ endsWith(matrix.target, '-musl') && 'musllinux_1_2' || (runner.os == 'Linux' && matrix.architecture == 'aarch64' && '2_24' || 'auto') }}
```

Comment:
Missing brackets? I believe `runner.os == 'Linux' && matrix.architecture == 'aarch64'` should be enclosed in extra brackets:

```yaml
manylinux: ${{ endsWith(matrix.target, '-musl') && 'musllinux_1_2' || ((runner.os == 'Linux' && matrix.architecture == 'aarch64') && '2_24' || 'auto') }}
```